.travis/environments/travis/proxy.yml

```diff
@@ -1,2 +1,3 @@
 SITE_HOST: '{{ groups.proxy.0 }}'
 fake_ssl_cert: yes
+nginx_worker_rlimit_nofile: 65536
```

Comment:
what's this?

Reply:
I did it in this commit: https://github.com/dimagi/commcare-cloud/pull/2013/commits/0bc45f18f1830c57c00c0c8cf51133b45a5b9074; at some point I ran the nosetests directly against the travis environment, and it complained that `nginx_worker_rlimit_nofile` is supposed to live in `proxy.yml`. I looked at our other environments, and this is closer to what it looked like (with the number here hardcoded), so I decided to follow that example.